gitlab/resource_gitlab_managed_license_test.go

```diff
@@ -0,0 +1,168 @@
+package gitlab
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/xanzy/go-gitlab"
+	"strconv"
+	"testing"
+)
+
+func TestAccGitlabManagedLicense_basic(t *testing.T) {
+	var managedLicense gitlab.ManagedLicense
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() {},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckManagedLicenseDestroy,
+		Steps: []resource.TestStep{
+			{
+				// Create a managed license with an "approved" state
+				SkipFunc: isRunningInCE,
+				Config:   testManagedLicenseConfig(rInt, "approved"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGitlabManagedLicenseExists("gitlab_managed_license.fixme", &managedLicense),
+				),
+			},
+			{
+				// Update the managed license to have a blacklisted state
+				SkipFunc: isRunningInCE,
+				Config:   testManagedLicenseConfig(rInt, "blacklisted"),
```

Comment:
For the "update" TestStep you should provide different config than the first step.

Reply:
Hey @armsnyder  - The config is different, it updates from the status from "approved" to "blacklisted", which is why the check step for the second test checks the status as opposed to just that it exists :)

Reply:
🤦 ahhh that's right, my bad